README.md

```diff
@@ -114,7 +114,7 @@ It also works with full URLs like
 Use `npm run debug:server` to start server in debug mode.
 [This recipe][nodemon debug] shows how to debug Node.js application in [VS Code][].
 
-Shields has experimental support for [Gitpod Beta][gitpod], a pre-configured development
+Shields has experimental support for [Gitpod][gitpod], a pre-configured development
 environment that runs in your browser. To use Gitpod, click the button below and
 sign in with GitHub. Gitpod also offers a browser add-on, though it is not required.
 Please report any Gitpod bugs, questions, or suggestions in issue
```

Comment:
I know this line was already in here, but wouldn't it be better to have folks direct their Gitpod related bugs/questions/etc. to Gitpod?

I know I wouldn't be the best person to help folks out with Gitpod related questions/issues/etc. and I suspect most of the other Shields maintainers don't have Gitpod expertise either.

Reply:
I think that's kind of the goal of that issue, is to funnel people to one place to get help.

Though I think it's true that none of the Shields maintainers are very familiar with Gitpod, and while as a collective we're well-equipped to support Shields in various environments, we're not well-equipped to support Gitpod.

Jan, I wonder if you could clarify what level of support you and/or Gitpod are available to respond to problems people might run into.

We haven't exactly been crushing under a load of support requests; in fact I haven't seen any. Since you say people are using it, they are either managing okay, or not raising issues in any of the Shields forums. Though I do get where the core team is coming from, in setting expectations that this is something where we will need your help to provide support.

Reply:
Thanks a lot for the additional feedback!

> wouldn't it be better to have folks direct their Gitpod related bugs/questions/etc. to Gitpod?

That's an excellent point. And @paulmelnikow is right: that's indeed the goal behind issue #2772. I'm subscribed to it, and will see / be able to reply to any comment posted there.

> Jan, I wonder if you could clarify what level of support you and/or Gitpod are available to respond to problems people might run into.

Sure! So for Gitpod-related problems or questions in Shields, please feel free to redirect anyone to #2772, or to @mention me directly in any discussion where you feel my input could be helpful. I'm happy to spend both working hours and personal time on helping out Shields developers with Gitpod. 🙂

> Since you say people are using it, they are either managing okay, or not raising issues in any of the Shields forums.

I do hope it's the former! 🤞 We've put significant effort into the Shields/Gitpod dev experience. But if you notice anyone struggling with / being annoyed by something that looks somewhat Gitpod-related, please ping me if possible. 🙏